SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -314,8 +332,14 @@ void ActorSystem::Advance()
 		ProcessAdds(SubView);
 	}
 
+	// Order here matters: Rep Notifies should be called before authority gains
```

Comment:
Can you expand this comment to explain why?

Reply:
added

```
	// Order here matters: Rep Notifies should be called before authority gains.
	// This is because it wouldn't make sense to be told a property was updated while you are authoritative over that actor, as while
	// authoritative you are supposed to be the only server that can update properties.
```